server/datastore/mysql/microsoft_mdm.go

```diff
@@ -633,17 +633,20 @@ WHERE
 
 func (ds *Datastore) GetMDMWindowsBitLockerStatus(ctx context.Context, host *fleet.Host) (*fleet.HostMDMDiskEncryption, error) {
 	if host == nil {
-		return nil, errors.New("host cannot be nil")
+		return nil, errors.New("cannot get bitlocker status for nil host")
```

Comment:
Would be best to use `ctxerr.New` here (and `ctxerr.Wrap[f]` elsewhere in this method), so we properly capture the information.